src/app/components/ManualEntry.svelte

```diff
@@ -42,39 +83,46 @@
     box-shadow: 0 25px 40px 0 rgba(0, 0, 0, 0.1);
   }
 
-  tr {
+  table th {
+    vertical-align: middle;
+    padding: 1.25rem 0 1.25rem 1.25rem;
+    background-color: var(--lite-white);
+    text-align: center;
+  }
+
+  table tr {
     border-bottom: 1pt solid var(--lite-white);
   }
 
-  td {
+  table td {
     vertical-align: middle;
     padding: 0.625rem 0 0.625rem 1.25rem;
     color: var(--gray-dark);
     text-align: center;
```

Comment:
I think center alignment should be avoided. [See this answer on UX stack exchange](https://ux.stackexchange.com/a/113594). The correct alignment depends on the nature of the content, but I think for words it should be left aligned for LTR langauges (e.g., English, Chinese) and right-aligned for RTL languages (e.g., Arabic Hebrew). You can use the `text-align: start` for this:

```suggestion
    text-align: start;
```

Arabic numbers (i.e., 1234567890) should _always_ be right-aligned.